colossalai/inference/hybridengine/engine.py

```diff
@@ -8,31 +9,35 @@
 from colossalai.shardformer import ShardConfig, ShardFormer
 from colossalai.shardformer.policies.base_policy import Policy
 
+from ..pipeline.microbatch_manager import MicroBatchManager
 from ..tensor_parallel.kvcache_manager import MemoryManager
-from .microbatch_manager import MicroBatchManager
 
+PP_AXIS, TP_AXIS = 0, 1
 
-class PPInferEngine:
+_supported_models = [
+    "LlamaForCausalLM",
+]
+
+
+class CaiInferEngine:
     """
-    PPInferEngine is a class that handles the pipeline parallel inference.
+    InferEngine is a class that handles the pipeline parallel inference.
```

Comment:
Make the name consistent (`CaiInferEngine` or `InferEngine`)

Reply:
yes sir!